src/app/_utils/structuredData.ts

```diff
@@ -17,13 +27,32 @@ export const baseOrganizationSchema: WithContext<Organization> = {
   '@type': 'Organization',
   name: ORGANIZATION_NAME,
   url: BASE_URL,
+  sameAs: [
+    'https://bsky.app/profile/ff.bsky.social',
+    'https://github.com/FilecoinFoundationWeb',
+    'https://www.linkedin.com/company/filecoin-foundation/mycompany/',
+    'https://twitter.com/FilFoundation',
+    'https://www.youtube.com/channel/UCeM5ezNgFM1Sle_gIz-KMMA',
+  ],
   description: seo.description,
   logo: {
     '@type': 'ImageObject',
     url: '', // Replace with actual logo URL
     width: '600', // Replace with actual width
     height: '60', // Replace with actual height
   },
+  contactPoint: [
+    {
+      '@type': 'ContactPoint',
+      contactType: 'Media and collaboration inquiries',
+      email: 'mailto:hello@fil.org',
+    },
+    {
+      '@type': 'ContactPoint',
+      contactType: 'Ecosystem grants inquiries',
+      email: 'mailto:devgrants@fil.org',
+    },
+  ],
```

Comment:
Same - Let's use `FILECOIN_FOUNDATION_URLS`

Reply:
We can make `email` an object with a label and whatever we have above under `contactType` - Please refactor the rest as well (No need to use label when it's different.)

Reply:
do you mean something like this? 

```
 grants: {
    github: 'https://github.com/filecoin-project/devgrants',
    email: {
      label: 'Ecosystem grants inquiries',
      address: 'mailto:devgrants@fil.org',
    },
  },
```

Reply:
Exactly! The only thing I would change is to rename `address` to `href`